WordleGame.java

```diff
@@ -0,0 +1,90 @@
+
+import java.io.*;
+import java.util.*;
+
+public class WordleGame {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in); //scanner for user input
+        System.out.println("Welcome to Wordle!");
+        System.out.println();
+
+        //read in dictionary
+        Set<String> dictionary = new HashSet<>();
+        try {
+            Scanner fileScanner = new Scanner(new File("dictionary.txt")); // scanner for dictionary
+            while (fileScanner.hasNextLine()) {
+                String word = fileScanner.nextLine().trim().toLowerCase();
+                if (word.length() == 5) {
+                    dictionary.add(word);
+                }
+            }
+            fileScanner.close();
+        } catch (FileNotFoundException e) {
+            System.out.println("Error: dictionary.txt file not found.");
+            return;
+        }
+
+
+        // picking a random word
+        List<String> wordList = new ArrayList<>(dictionary);
```

Comment:
You don't need to create a whole extra data structure to pick a random word. You could choose a random number and iterate that number of times into the Set